src/Keboola/StorageApi/Client.php

```diff
@@ -2217,7 +2222,7 @@ protected function request($method, $url, $options = array(), $responseFileName
             if (!$responseFile) {
                 throw new ClientException("Cannot open file {$responseFileName}");
             }
-            $body = $response->getBody();
+            $body = $this->getBody($response);
```

Comment:
Tady vracíš string... ale o řádek níž na něm voláš ->eof()... takže takhle je ta implementace taková trochu nebezpečná...

Reply:
Ideálně něco jako je v \GuzzleHttp\Handler\StreamHandler::checkDecode

Reply:
Aby se to chovalo furt stejně... jen to mělo dekódované contents

Reply:
Ha, ono se vlastne pocita se streamem, upravim.

Reply:
vyreseno pomoci `BrotliStream`